tasks/queue-workflow/schemas/input.json

```diff
@@ -2,15 +2,8 @@
   "title": "QueueWorkflowInput",
   "description": "Describes the input expected by the queue-workflow task",
   "type": "object",
-  "required": ["workflow", "workflowInput"],
   "properties": {
```

Comment:
I didn't want to make it a required input because it's a required part of the config. So basically it _can_ be specified as an input if you'd like, but otherwise it should also fallback to the config value, which is require. Let me know if that doesn't make any sense